### PR TITLE
XW-1812 | track footer clicks

### PR DESCRIPTION
**Pull request by Warkot**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1812

## Description

We need to know the amount of clicks on the Full site link, so why not to check all links in current footer?

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/components/wikia-footer.js [hunk 1]
```diff
@@ -61,6 +62,11 @@ export default Ember.Component.extend({
 		 * @returns {void}
 		 */
 		handleFooterLinkClick(text, href) {
+			track({
```
Comment (rogatty) [inline front/main/app/components/wikia-footer.js]: Don't we need to add some delay to let tracking request being sent before opening new page?

File: front/main/app/components/wikia-footer.js [hunk 2]
```diff
@@ -61,6 +62,11 @@ export default Ember.Component.extend({
...
+				action: trackActions.click,
+				category: text
```
Comment (kvas-damian) [inline front/main/app/components/wikia-footer.js]: What do you think about putting `text` into label and create category that has name related to footer?

Comment (kvas-damian) [inline front/main/app/components/wikia-footer.js]: BTW using text will be really hard to analyze when users start using it in different languages

Comment (Warkot) [inline front/main/app/components/wikia-footer.js]: Would `category: 'footer'` be ok?

Comment (Warkot) [inline front/main/app/components/wikia-footer.js]: `text` is actually the message key—you can see it in the file few lines above.

Comment (kvas-damian) [inline front/main/app/components/wikia-footer.js]: `footer` sounds good to me.
:+1: for `text`

File: front/main/app/components/wikia-footer.js (hunk 1 above)
Comment (Warkot) [inline front/main/app/components/wikia-footer.js]: What do you think now? Or maybe I should move the `later` logic to the `track` function, as where you pointed me in https://github.com/Wikia/app/blob/dev/resources/wikia/modules/tracker.js#L318-L330?

**Comment by rogatty** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**